### DISCUSSION TITLE
[Feature request] OMI/TMI support is very important feature for research.

### DISCUSSION THREAD

**Discussion by xinlnix** [Xyce/Xyce#84] [General]
Open Model Interface (CMC model building open source) which is based on TMI 2.01 (TSMC model interface). I've checked TMI is supported by two major simulator (HSPICE and Spectre) but not any other simulator.

**Comment by Karlsefni2012** [comment]
We are aware of OMI and it is part of our future plans.  However, it isn't our highest priority at the moment so the timeline for implementation is somewhat TBD.

**Comment by xinlnix** [comment]
Where can I find the TODO list of the project?
